test/test-span-data.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Is it necessary to change the copyright?

Reply:
The file contents were completely changed... enough so that I would consider it a new file.

Reply:
Ok sounds good.